waddrmgr/manager.go

```diff
@@ -1138,6 +1152,25 @@ func (m *Manager) Lock() error {
 	return nil
 }
 
+// LookupAccount loads account number stored in the manager for the given
+// account name
+func (m *Manager) LookupAccount(name string) (uint32, error) {
+	m.mtx.RLock()
+	defer m.mtx.RUnlock()
+
+	return m.lookupAccount(name)
+}
+
+func (m *Manager) lookupAccount(name string) (uint32, error) {
```

Comment:
Please define functions before they're called to stay consistent. Also, there is no comment.